src/main/kotlin/io/github/mojira/arisa/modules/commands/FixCapitalizationCommand.kt

```diff
@@ -9,13 +9,23 @@ class FixCapitalizationCommand : Command1<String> {
                 """(?<=\.\s|^|!\s|\?\s|\n)[A-Z][A-Za-z-'0-9]*((\s|,\s|;\s|:\s)[A-Z][A-Za-z-'0-9]*)*(?=\.|${'$'}|!|\?|\n)"""
                     .toRegex()
 
+        val exceptions = listOf(
+            "i",
+            "minecraft",
+            "mojang"
+        )
+
         var newDescription = issue.description!!
         val matchesDescription = capitalizationRegex.findAll(newDescription)
         matchesDescription
                 .map { it.groupValues[0] }
                 .forEach {
                     newDescription = newDescription.replace(it, it.toLowerCase().capitalize())
                 }
+        exceptions
+                .forEach {
+                    newDescription = newDescription.replace(" $it ", " ${it.capitalize()} ")
```

Comment:
I'm not actually sure if this would work

```suggestion
                    newDescription = newDescription.replace("\\b$it\\b".toRegex(), " ${it.capitalize()} ")
```

Reply:
Your change didn't seem to work